hack/lib/common.sh

```diff
@@ -117,6 +117,7 @@ function setup_envs {
   export TEST_ASSET_KUBECTL="$1"/bin/kubectl
   export TEST_ASSET_KUBE_APISERVER="$1"/bin/kube-apiserver
   export TEST_ASSET_ETCD="$1"/bin/etcd
+  export KUBE_RBAC_PROXY_IMAGE=gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
```

Comment:
This is static so it doesn't need to be dynamically exported. Set this as a global variable at the top of this file.

Reply:
Done 👍